holoviews/__init__.py

```diff
@@ -25,6 +25,7 @@
 from .operation import ElementOperation, MapOperation, TreeOperation # noqa (API import)
 from .element import *                                               # noqa (API import)
 from . import util # noqa (API import)
+from .element import __all__ as elements_list
```

Comment:
The problem is that ``__all__`` contains strings (which I want) but doing tighter subclass checks requires a list of those classes. I could iterate over ``locals`` but I find that pretty ugly. What would you suggest?